firmware/SetPath.cmd

```diff
@@ -0,0 +1,6 @@
+@set ARDUINO_DIRECTORY=E:\Elektro\arduino
```

Comment:
This is a path that is only valid on your machine.
Why is this file needed at all?

Reply:
For convenience for the valued volunteer, to enable using its Arduino installation for building. AFAIK there is no default installation directory, so I choosed my own.
If you have an idea, which directory path is suited better, please tell.

Reply:
hm.. I guess most people who know their way around V-USB also have a separate installation of AVR-GCC.  I think this will cause serious confusion.

Reply:
So I am no people which know way around V-USB. Its OK 😀
If you find SetPath.cmd confusing, should I remove it completely?

Reply:
Er... :) Well, I have a script that opens a shell with the right path. It's basically the same, just not part of the MN repository.

I think it should be removed from the repository, because people will have different preferences how to configure their build enviroment.

Reply:
Done. Waiting for commit...

Reply:
It's not added to this pull request?

Reply:
Remove commited